documentation/sphinx/source/release-notes.rst

```diff
@@ -1,214 +1,58 @@
-* Fix: various objects internal to the bindings were not properly closed. <rdar://problem/35541447>
-
-Other Changes
--------------
-
-* Backups made prior to 5.1 can no longer be restored. <rdar://problem/25512772>
-* Backup now uses a hostname in the connection string instead of a list of IPs when backing up to blob storage. This hostname is resolved using DNS. <rdar://problem/34093405> 
-* ``fdbblob`` functionality has been moved to ``fdbbackup``. <rdar://problem/25512772>
-* ``fdbcli`` will warn the user if it is used to connect to an incompatible cluster. <rdar://problem/33363571>
-* Cluster files that do not match the current connection string are no longer corrected automatically. <rdar://problem/35129575>
-* Improved computation of available memory on pre-3.14 kernels. <rdar://problem/35336487>
-* Stopped reporting blob storage connection credentials in ``fdbbackup`` status output. <rdar://problem/31483629>
-
-Earlier release notes
----------------------
-* :doc:`5.0 (API Version 500) </old-release-notes/release-notes-500>`
-* :doc:`4.6 (API Version 460) </old-release-notes/release-notes-460>`
-* :doc:`4.5 (API Version 450) </old-release-notes/release-notes-450>`
-* :doc:`4.4 (API Version 440) </old-release-notes/release-notes-440>`
-* :doc:`4.3 (API Version 430) </old-release-notes/release-notes-430>`
-* :doc:`4.2 (API Version 420) </old-release-notes/release-notes-420>`
-* :doc:`4.1 (API Version 410) </old-release-notes/release-notes-410>`
-* :doc:`4.0 (API Version 400) </old-release-notes/release-notes-400>`
-* :doc:`3.0 (API Version 300) </old-release-notes/release-notes-300>`
-* :doc:`2.0 (API Version 200) </old-release-notes/release-notes-200>`
-* :doc:`1.0 (API Version 100) </old-release-notes/release-notes-100>`
-* :doc:`Beta 3 (API Version 23) </old-release-notes/release-notes-023>`
-* :doc:`Beta 2 (API Version 22) </old-release-notes/release-notes-022>`
-* :doc:`Beta 1 (API Version 21) </old-release-notes/release-notes-021>`
-* :doc:`Alpha 6 (API Version 16) </old-release-notes/release-notes-016>`
-* :doc:`Alpha 5 (API Version 14) </old-release-notes/release-notes-014>`
+#############
+Release Notes
+#############
+
+5.2.0
+=====
+
+Features
+--------
+
+* Backup and DR share a single mutation log when both are being used on the same cluster. Ongoing backups will be aborted when upgrading to 5.2. `(PR #3) <https://github.com/apple/foundationdb/pull/3>`_
+* Added a TLS plugin implementation. `(PR #343) <https://github.com/apple/foundationdb/pull/343>`_
+* Backup supports HTTPS for blobstore connections. `(PR #343) <https://github.com/apple/foundationdb/pull/343>`_
+* Added the APPEND_IF_FITS atomic operation. `(PR #22) <https://github.com/apple/foundationdb/pull/22>`_
+* Updated the SET_VERSION_STAMP_VALUE atomic operation to place the version stamp at a specified offset in a value. `(Issue #148) <https://github.com/apple/foundationdb/issues/148>`_
```

Comment:
It looks like the option is `SET_VERSIONSTAMPED_VALUE`, and we typically spell versionstamp without spaces.